recipes/pyorbital/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "pyorbital" %}
+{% set version = "1.3.1" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "6a5b4b31f83d1fefbc7759f968dcebe331cbe0d3e0a365ed29f37247b3e8a111" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
```

Comment:
Can this be noarch?

Reply:
Done. `noarch: python`, let me know if that isn't the proper way anymore.